magda-migrator-registry-db/sql/V2__Support_multi_tenants.sql

```diff
@@ -0,0 +1,49 @@
+CREATE SEQUENCE IF NOT EXISTS tenants_id_seq;
+
+CREATE TABLE IF NOT EXISTS Tenants (
```

Comment:
Yeah the lack of foreign keys is regrettable but one of the tradeoffs we have to make when using microservices :(. Thing is while the foreign keys help with the registry, we'll still have to add a tenant column to the `content-db`, `authorization-db` etc so it can only ever help us so much.

Reply:
Done.